packages/wallets/src/evm/wallets/coinbase-wallet.ts

```diff
@@ -42,7 +40,7 @@ export class CoinbaseWallet extends AbstractBrowserWallet<{
         options: {
           appName: this.options.dappMetadata.name,
           reloadOnDisconnect: false,
-          darkMode: this.options.theme === "dark",
+          darkMode: this.options.dappMetadata.isDarkMode,
```

Comment:
darkMode is specific to CoinbaseWallet in web, what do you think if we remove it from the dappMetadata and pass it as a  special option for CoinbaseWallet instead?

Reply:
I thought Wallet Connect has color schemes, too?